js/block.js

```diff
@@ -269,7 +269,8 @@ class Block {
                     if (counter !== undefined) {
                         loopCount = counter;
                     }
-                    if (loopCount > 10) {  // race condition?
```

Comment:
All of these formatting changes are fine but not really relevant to this MR.

Reply:
Sir actually in my vs code I have an extension that auto-formats the document after saving. Shall I revert these changes?